FNA.Core.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+		<TargetFrameworks>net50</TargetFrameworks>
```

Comment:
Are we able to keep all three of these in a single csproj? I thought the newer compilers defined stuff like NET5_0 by default.